docs/UserGuide.md

```diff
@@ -44,29 +44,30 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in diamond brackets `<>` are parameters to be supplied by the user.<br>
+  e.g. in `new n/<name>`, `<name>` is a parameter which can be used as `new n/John`.
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+* Parameters with ... can be used multiple times.<br>
+  e.g. in  `new n/<name> [t/<tag>]…`, `[t/<tag>]…` is a parameter which can be used as `new n/John t/forgetful t/loser`
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Items in square brackets `[]` are optional.<br>
+  e.g. in `new n/<name> [t/<tag>]…` can be used as `new n/Alan Poe t/shy pooper` or as `new n/Alan Poe`.
```

Comment:
Might want to use "add" instead of "new" as the example commands as we do not have a "new" command